Installation - physical fails on pi

using this on the pi: ```curl https://raw.githubusercontent.com/andrewn/neue-radio/master/deployment/provision | sudo bash```

```
Update the following: debug, downloader, manager, physical, rfid, setup, speech, radio, websocket

Completed: debug
Completed: websocket
Completed: rfid
Completed: radio
Completed: speech
Completed: setup
Completed: manager
Completed: downloader
Error installing/updating: physical

1 failure.
```

and also ```npm install-all``` on the mac.

```cd /opt/radiodan/rde/services/physical && npm install
npm WARN physical@0.0.1 No description
npm WARN physical@0.0.1 No repository field.

npm ERR! path /opt/radiodan/rde/services/physical/node_modules/.staging
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall rmdir
npm ERR! Error: EACCES: permission denied, rmdir '/opt/radiodan/rde/services/physical/node_modules/.staging'
npm ERR!  { Error: EACCES: permission denied, rmdir '/opt/radiodan/rde/services/physical/node_modules/.staging'
npm ERR!   stack: 'Error: EACCES: permission denied, rmdir \'/opt/radiodan/rde/services/physical/node_modules/.staging\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'rmdir',
npm ERR!   path: '/opt/radiodan/rde/services/physical/node_modules/.staging' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2018-03-28T10_50_46_884Z-debug.log```

--- a/deployment/update
+++ b/deployment/update
@@ -39,7 +39,7 @@ const findApplicableDirectories = async rootDir => {
 
 const install = async ({ name, path }) => {
   try {
-    await exec(`npm install --production`, {
+    await exec(`npm install --unsafe-perm --production`, {
       cwd: path,
       env: { ...process.env, JOBS: "max" }
     });

